proposals/spi-proposal.md

```diff
@@ -267,13 +252,13 @@ N:  END_SYSEX
 
 ### SPI_END
```

Comment:
While SPI_BEGIN may not be necessary, I'm not sure yet if SPI_END is useful. I could see a need to provide this in order to clean up when closing the user application, but that may also depend on the particular platform this proposal is implemented for.

Reply:
On Linux SPI_END could be used to free resources, for example, to close files. The being said, if the application is about to terminate lots of resources will be freed automatically by Linux.